code/modules/paperwork/paper.dm

```diff
@@ -72,7 +72,7 @@
 
 /obj/item/paper/Initialize(mapload)
 	. = ..()
-	AddComponent(/datum/component/customizable_reagent_holder, /obj/item/clothing/mask/cigarette/rollie, CUSTOM_INGREDIENT_ICON_NOCHANGE, ingredient_type=CUSTOM_INGREDIENT_TYPE_DRYABLE, max_ingredients=2, job_xp = 1, job = JOB_BOTANIST)
+	//AddComponent(/datum/component/customizable_reagent_holder, /obj/item/clothing/mask/cigarette/rollie, CUSTOM_INGREDIENT_ICON_NOCHANGE, ingredient_type=CUSTOM_INGREDIENT_TYPE_DRYABLE, max_ingredients=2, job_xp = 1, job = JOB_BOTANIST) //MONKESTATION REMOVAL: fix adding C4 on paper
```

Comment:
issue with this is that it breaks making blunts

Reply:
Obj/item/rollingpaper, printable from biogens, still exists which this doesnt effect